languagetool-language-modules/eo/src/main/resources/org/languagetool/rules/eo/grammar.xml

```diff
@@ -4086,15 +4128,16 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     </rulegroup>
     <rule id="NOVA_ZELANDO" name="Nova-Zelando → Nov-Zelando">
       <pattern>
-        <token regexp="yes">Nova-?(?:Jork|Kaledoni|Orlean|Zeland)on?</token>
+        <token regexp="yes">Nov(a|o)-?(?:Amsterdam|Angl(i|uj)|Brunsvik|Delhi|Gvine|Hampŝir|Holand|Jork|Jorki|Kaledoni|Meksik|Orlean|Skot(i|land|uj)|Zeland|Zeml|Ĵerzej)on?</token>
       </pattern>
       <message>Oni devas skribi <suggestion>Nov-Zelando</suggestion> aŭ <suggestion>Nova Zelando</suggestion>.</message>
```

Comment:
There is an error in this line, between the suggestion tags there should be a regular expression.
This screenshot comes directly from https://languagetool.org/:
![image](https://user-images.githubusercontent.com/692029/35914658-574ca5e4-0c04-11e8-8645-f9fc799f7776.png)